tasks/install_package.yml

```diff
@@ -28,6 +28,13 @@
   changed_when:
     _conga_aem_packages_wcmio_content_package_maven_plugin_version != conga_aem_packages_wcmio_content_package_maven_plugin_version
 
+- name: "Use interactive mode for '{{ item.path | basename }}'"
+  pause:
+    prompt: "Continue with deploying {{ item.path | basename }}?"
+  when:
+    - conga_aem_packages_interactive_packages is defined
+    - (item.path | basename) in conga_aem_packages_interactive_packages
```

Comment:
Would be nice to allow regex based matching here so that you don't have to exactly specify the package name, wouldn't it?

Reply:
I tried this but i did not get this to work nicely since you have to check if a list of regular expressions would match to the current item. I postponed this.